src/analysis/classic/hunter/lowRankSpells.ts

```diff
@@ -2,7 +2,12 @@ import SPELLS from 'common/SPELLS/classic/hunter';
 
 const lowRankSpells = Object.entries(SPELLS).reduce((result, [str, obj]) => {
   if (obj.lowRanks) {
-    Object.assign(result, { [obj.id]: [...obj.lowRanks] });
+    // handle rank 4 and rank 3 low level spell allowed/encouraged in WoTLK
```

Comment:
If you want to mark certain spells as "ok to use", see the whitelist example in the warlocks folder.

Reply:
That works better. 
Done